### PR TITLE
Add `prettier-plugin-solidity` dev dependency v2

### LINKED ISSUE #4994: Prettier: `No parser could be inferred for file` Error for Solidity Files
Receiving errors:

```bash
web3-providers-http: [error] No parser could be inferred for file: test/shared_fixtures/sources/Basic.sol
```

when running `yarn format`

### PR DESCRIPTION
**Pull request by Spacesai1or**
closes #4994

### CODE REVIEW

File: packages/web3-eth-contract/test/integration/conract_deploy.test.ts [hunk 1]
```diff
@@ -132,7 +132,7 @@ describe('contract', () => {
 			).rejects.toThrow('Returned error: intrinsic gas too low');
 		});
 
-		it('should fail with errors deploying a zero length bytecode', () => {
+		it.skip('should fail with errors deploying a zero length bytecode', () => {
```
Comment (avkos) [inline packages/web3-eth-contract/test/integration/conract_deploy.test.ts]: Why do you skip this test?

**Review by avkos — CHANGES_REQUESTED** [review]

File: packages/web3-eth-contract/test/integration/conract_deploy.test.ts (hunk 1 above)
Comment (nazarhussain) [reply packages/web3-eth-contract/test/integration/conract_deploy.test.ts]: Yes I fixed this one in my recent PR. This should not be skipped.

**Review by nazarhussain — APPROVED** [review]
Other than @avkos  comments, rest  LGTM.

**Review by jdevcs — APPROVED** [review]

**Review by luu-alex — APPROVED** [review]

File: packages/web3-eth-contract/test/integration/conract_deploy.test.ts (hunk 1 above)
Comment (Spacesai1or) [reply packages/web3-eth-contract/test/integration/conract_deploy.test.ts]: @nazarhussain Can I just remove the `skip`? This branch is up to date with `4.x`, so not sure how this snuck in here

Comment (nazarhussain) [reply packages/web3-eth-contract/test/integration/conract_deploy.test.ts]: @spacesailor24 Yes you can remove the skip.

Comment (Spacesai1or) [inline packages/web3-eth-contract/test/integration/conract_deploy.test.ts]:
```suggestion
		it('should fail with errors deploying a zero length bytecode', () => {
```

Comment (Spacesai1or) [reply packages/web3-eth-contract/test/integration/conract_deploy.test.ts]: It's been removed

**Review by avkos — APPROVED** [review]

**Pull request merged**